ops/Makefile

```diff
@@ -43,8 +43,8 @@ api.tfvars: /dev/null
 	echo "deploy_actor=\"$(GITHUB_ACTOR)\"" >> $@;
 
 init-%: .valid-env-%
-	terraform -chdir=$*/persistent init
-	terraform -chdir=$* init
+	terraform -chdir=$*/persistent init -upgrade
```

Comment:
Oops, I missed this when I was going through on my last review...should these `-upgrade` flags (line 46 and 47) now be removed, since the lock files have been manually updated?

Reply:
Yes, another good catch - just removed these